envoy/config/filter/http/jwt_auth/v2/BUILD

```diff
@@ -0,0 +1,9 @@
+licenses(["notice"])  # Apache 2
+
+load("//bazel:api_build_system.bzl", "api_proto_library")
+
+api_proto_library(
+    name = "jwt_auth",
```

Comment:
Consider renaming "jwt_auth" to "jwt_authn" if it is for authentication.

Reply:
Done